piksi_tools/console/console.py

```diff
@@ -715,11 +710,16 @@ def update_serial():
                 if self.networking_view.connect_when_uuid_received:
                     self.networking_view._connect_rover_fired()
 
+            skip_settings_read = False
+            if 'mode' in self.connection_info:
+                if self.connection_info['mode'] == 'file':
+                    skip_settings_read = True
+
```

Comment:
those errors when testing from a sbp file always bothered me; thanks for fixing.